src/singularity.ts

```diff
@@ -136,20 +136,31 @@ export class SingularityUpgrade extends DynamicUpgrade {
     public async buyLevel(event: MouseEvent): Promise<void> {
         let purchased = 0;
         let maxPurchasable = 1
+        player.goldenQuarks = 1e100
```

Comment:
```suggestion

```

Reply:
It seems that I forgot to erase the garbage in the test. Forgive me.